clients/osv/src/test/kotlin/ModelTest.kt

```diff
@@ -41,7 +41,7 @@ class ModelTest : StringSpec({
     }
 })
 
-private val JSON = Json {}
+private val JSON = Json.Default
```

Comment:
I'd prefer to have this inlined now that this doesn't create a new instance.

Reply:
Would you agree moving the JSON constant from ´OsvApiClient` to `Model.kt` ?

Reply:
Actually no, I would not agree, because IMO the model should not know anything about its concrete representation in a file format.